assignments.md

```diff
@@ -10,31 +10,3 @@ The lab computers are Falcon Northwest PCs with dual 27-inch monitors, NVidia Ge
 If you choose to develop outside the lab, you should test and experience your work at least once on the lab computers, as we will also grade your work there.
 
 Unfortunately, we have only 20 workstations and over 200 students. Therefore, you should start early on assignments and work in the lab during less popular times. Unless announced otherwise, the lab is available 24/7.
```

Comment:
We used to make a big mega-post at the beginning of the semester with tips on how to do well in the course, which has since been lost to history. [I pasted the text here](https://pastebin.com/Kvfp6Umm), it might be worth dropping that stuff in here.

Reply:
Good advice and fun to read. Just need to edit some of the outdated stuff in it

Reply:
It was Mike's doing :joy: